src/main/java/bouncybob/task/ToDos.java

```diff
@@ -0,0 +1,26 @@
+package bouncybob.task;
+
+/**
+ * Represents a To-Do task in the BouncyBob application.
+ */
+public class ToDos extends Task {
```

Comment:
Perhaps ToDo instead of ToDos since it represents a single task? Same for the other task classes.